flink-dist/pom.xml

```diff
@@ -113,8 +113,13 @@ under the License.
 			<artifactId>flink-metrics-jmx</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+        
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-mesos_2.10</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
Thought about it but I don't see a good reason.   Why that and not for the various connectors and other such non-core modules?  I see Kinesis is special cased but for licensing reasons.